servicetalk-loadbalancer-experimental-provider/src/main/java/io/servicetalk/loadbalancer/experimental/DefaultLoadBalancerObserver.java

```diff
@@ -68,7 +68,8 @@ public void onHostSetChanged(Collection<? extends Host> newHosts) {
                     healthyCount++;
                 }
             }
-            LOGGER.debug("{}- onHostSetChanged(total: {}, healthy: {})", lbDescription, newHosts.size(), healthyCount);
+            LOGGER.debug("{}- onHostSetChanged(newHosts: {}). Size: {}, healthy: ", lbDescription, newHosts,
+                    newHosts.size(), healthyCount);
```

Comment:
I would propose to add collection at the end. Otherwise, it might be hard to see what is the size and `healthyCount` if these numbers come after a long collection.